src/utils.ts

```diff
@@ -1,7 +1,8 @@
-import * as amplitude from "@amplitude/analytics-node";
 import { faker } from "@faker-js/faker";
 
-const createFakeUserProperties = () => {
+export const API_KEY = "80298077b0b863e8ef2b3e8351b2dfd3";
```

Comment:
Should we hide this API key since it's public? add a note?

Reply:
oh shoot, thanks for catching

Reply:
removed